mrtt-ui/src/components/RestorationAimsForm/RestorationAimsForm.js

```diff
@@ -1,21 +1,46 @@
 import { FormLabel } from '@mui/material'
-import { useState } from 'react'
 import { useForm } from 'react-hook-form'
-import { FormQuestionDiv, MainFormDiv, SectionFormTitle } from '../../styles/forms'
+import { useState } from 'react'
+import axios from 'axios'
+
+import { ErrorText } from '../../styles/typography'
+import { Form, FormQuestionDiv, MainFormDiv, SectionFormTitle } from '../../styles/forms'
 import ButtonSubmit from '../ButtonSubmit'
 import CheckboxGroupMangroveWithLabel from '../CheckboxGroupMangroveWithLabel'
+import language from '../../language'
 import restorationAimsQuestionsAndAnswers from './restorationAimsQuestionsAndAnswers'
 
+const formatFormDataForApi = (formData) =>
+  Object.entries(formData).map((formField) => {
+    const formAnswers = formField[1].map((formAnswer) => ({ choice: formAnswer }))
+
+    return {
+      question_id: `Q3.${formField[0]}`,
+      answer_value: formAnswers
+    }
+  })
```

Comment:
This is a pretty simple making-big-assumptions algorithm, but for now it works. It prob wont work when we mix in q3.4, but we'll cross that bridge when we get there.